lib/server.js

```diff
@@ -85,15 +86,31 @@ module.exports = function createServer(config, log) {
     throw err
   }
 
-  function fetchRecords(email, ip) {
-    return P.all(
-      [
-        // get records and ignore errors by returning a new record
-        mc.getAsync(email).then(EmailRecord.parse, EmailRecord.parse),
-        mc.getAsync(ip).then(IpRecord.parse, IpRecord.parse),
-        mc.getAsync(ip + email).then(IpEmailRecord.parse, IpEmailRecord.parse)
-      ]
-    )
+  function fetchRecords(email, ip, includeReputation) {
+    var promises = [
+      // get records and ignore errors by returning a new record
+      mc.getAsync(email).then(EmailRecord.parse, EmailRecord.parse),
+      mc.getAsync(ip).then(IpRecord.parse, IpRecord.parse),
+      mc.getAsync(ip + email).then(IpEmailRecord.parse, IpEmailRecord.parse)
+    ]
+    if (includeReputation) {
+      var fetchReputation = ipClient.get(ip)
```

Comment:
nit: it might be beneficial to extract this new`fetchReputation` logic into its own function.